vimeo-networking/src/main/java/com/vimeo/networking/model/User.java

```diff
@@ -179,6 +179,21 @@ public int getLikesCount() {
     }
 
     @Nullable
+    public Connection getChannelsConnection() {
+        if (metadata != null && metadata.connections != null && metadata.connections.channels != null) {
+            return metadata.connections.channels;
```

Comment:
These are the channels that a user follows. Is this what we want to display on the user profile?... because I could have sworn we only wanted to show channels that the user helped curate. There is a ```moderated_channels``` connection for this - I think you should add that as part of this PR in addition to this, it would be good to have both available on the User model.